R-packages/modeltools/R/response.R

```diff
@@ -0,0 +1,95 @@
+library(assertthat)
```

Comment:
There shouldn't be any `library()` calls in the code.

Reply:
Done

Reply:
More annoyances: we don't have CI, but it would fail (for the same reason that @JedGrabman's #509 is failing). Can you call:
```r
use this::use_package("assertthat")
```
or manually add `assertthat` to `DESCRIPTION`?